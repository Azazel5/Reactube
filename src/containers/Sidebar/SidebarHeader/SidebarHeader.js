import React from 'react'
import './SidebarHeader.scss'
import { Menu } from 'semantic-ui-react'

const sidebarHeader = (props) => {
    const heading = props.title ? props.title.toUpperCase(): ''
    return (
        <Menu.Item>
            <Menu.Header className="sidebar-header">{heading}</Menu.Header>
        </Menu.Item>
    )
}

export default sidebarHeader