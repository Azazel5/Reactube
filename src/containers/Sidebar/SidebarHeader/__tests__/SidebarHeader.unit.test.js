import React from 'react'
import {shallow} from 'enzyme'
import SidebarHeader from '../SidebarHeader'

describe('<SidebarHeader />', () => {
    test('renders <SidebarHeader /> with no props title', () => {
        const wrapper = shallow(<SidebarHeader />)
        expect(wrapper).toMatchSnapshot()
    })

    test('renders <SidebarHeader /> with no \'Sample Title\' title', () => {
        const wrapper = shallow(<SidebarHeader title='Sample Title'/>)
        expect(wrapper).toMatchSnapshot()
    })
})