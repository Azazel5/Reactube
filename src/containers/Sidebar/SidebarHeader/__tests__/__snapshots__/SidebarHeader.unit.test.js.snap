// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidebarHeader /> renders <SidebarHeader /> with no 'Sample Title' title 1`] = `
<MenuItem>
  <MenuHeader
    className="sidebar-header"
  >
    SAMPLE TITLE
  </MenuHeader>
</MenuItem>
`;

exports[`<SidebarHeader /> renders <SidebarHeader /> with no props title 1`] = `
<MenuItem>
  <MenuHeader
    className="sidebar-header"
  />
</MenuItem>
`;
